docs/plugins/ms-graph-authz.asciidoc

```diff
@@ -0,0 +1,112 @@
+[[ms-graph-authz]]
+=== Microsoft Graph Authz
+
+The Microsoft Graph Authz plugin uses https://learn.microsoft.com/en-us/graph/api/user-list-memberof[Microsoft Graph] to look up group membership information from Microsoft Entra ID.
+
+This is primarily intended to work around the Microsoft Entra ID maximum group size limit (see https://learn.microsoft.com/en-us/security/zero-trust/develop/configure-tokens-group-claims-app-roles#group-overages[Group overages]).
+
+:plugin_name: microsoft-graph-authz
+include::install_remove.asciidoc[]
```

Comment:
it would be nice to apply the same "how to install for other platforms" edits here.

also would be good to have the link down to the config pages on this page.